.github/workflows/scripts/update-docker-tags.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+CONSTRAINT=$1
+
+update-docker-tags \
+  -enforce="sourcegraph/cadvisor=$CONSTRAINT" \
+  -enforce="sourcegraph/frontend=$CONSTRAINT" \
+  -enforce="sourcegraph/jaeger-agent=$CONSTRAINT" \
+  -enforce="sourcegraph/github-proxy=$CONSTRAINT" \
+  -enforce="sourcegraph/gitserver=$CONSTRAINT" \
+  -enforce="sourcegraph/grafana=$CONSTRAINT" \
+  -enforce="sourcegraph/indexed-searcher=$CONSTRAINT" \
+  -enforce="sourcegraph/search-indexer=$CONSTRAINT" \
+  -enforce="sourcegraph/jaeger-all-in-one=$CONSTRAINT" \
+  -enforce="sourcegraph/postgres-11.4=$CONSTRAINT" \
+  -enforce="sourcegraph/precise-code-intel-bundle-manager=$CONSTRAINT" \
+  -enforce="sourcegraph/precise-code-intel-worker=$CONSTRAINT" \
+  -enforce="sourcegraph/prometheus=$CONSTRAINT" \
+  -enforce="sourcegraph/query-runner=$CONSTRAINT" \
+  -enforce="sourcegraph/redis-cache=$CONSTRAINT" \
+  -enforce="sourcegraph/redis-store=$CONSTRAINT" \
+  -enforce="sourcegraph/repo-updater=$CONSTRAINT" \
+  -enforce="sourcegraph/searcher=$CONSTRAINT" \
+  -enforce="sourcegraph/symbols=$CONSTRAINT" \
```

Comment:
Can we do `-enforce="sourcegraph/.*=$CONSTRAINT"` to cover them all without explicitly listing each?

Reply:
I had a stab at implementing this: https://github.com/bobheadlabs/update-docker-tags/commit/38ab3ab34faa97fa18b80a4d3ae76bde9c3d14e9

We have quite a few images that don't adhere to the 3.XX scheme (alpine, postgres, etc off the top of my head)